civitai-downloader/src/cli.py

```
@@ -0,0 +1,283 @@
+            except Exception as e:
+                click.echo(f"Download failed: {e}", err=True)
+    
+    asyncio.run(_download())
+
+
+@cli.command()
+@click.option('--limit', '-l', type=int, default=10, help='Number of items to show')
+@click.pass_context
+def list(ctx, limit: int):
+    """List download history."""
+    config = ctx.obj['config']
```

Ruff (0.12.2)

Local variable `config` is assigned to but never used

Remove assignment to unused variable `config`

(F841)